gpii/node_modules/processReporter/src/ProcessGet.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * GPII Process GET Handler
+ *
+ * Copyright 2015 OCAD University
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion");
+fluid.registerNamespace("gpii");
+
+fluid.defaults("kettle.requests.request.handler.processGet", {
```

Comment:
This file has been bound against an outdated version of universal and Infusion and needs to be updated to the current trunk